experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -1,8 +1,7 @@
 function basis_lie_highest_weight_compute(
-  L::LieAlgebra,
-  highest_weight::Vector{Int},
+  M::ModuleData,
   operators::Vector{RootSpaceElem},     # monomial x_i is corresponds to f_operators[i]
-  monomial_ordering_symb::Symbol,
+  monomial_ordering_symb::Symbol
```

Comment:
```suggestion
  monomial_ordering_symb::Symbol,
```
The formatter will add it back anyway